packages/kbn-ui-shared-deps/entry.js

```diff
@@ -44,6 +44,8 @@ export const Theme = require('./theme.ts');
 export const Lodash = require('lodash');
 export const LodashFp = require('lodash/fp');
 
+export const Fflate = require('fflate/esm/browser');
```

Comment:
@spalger suggestion - `kbn-ui-shared-deps` is minified more strongly, so it seems. 🤷🏻

Reply:
There are two reasons, first because of the minification available in ui-shared-deps and second because it allows us to rewrite the imports from `common` dirs to `fflate/esm/browser` but let them import the default implementation in node.

All that said, I didn't learn about the [DecompressionStream](https://developer.mozilla.org/en-US/docs/Web/API/DecompressionStream) API until yesterday so I'm wondering if maybe fflate is the best option or if we should be using this API which appears supported in all necessary browsers.

Reply:
Oh, seeing the other discussion in here it sounds like DecompressionStreams have already been ruled out